tests/dir.props

```diff
@@ -33,6 +33,22 @@
     <SkipImportILTargets>true</SkipImportILTargets>
   </PropertyGroup>
 
+  <!-- Common properties -->
+  <PropertyGroup>
+    <RootBinDir>$(__RootBinDir)\</RootBinDir>
+    <RootBinDir Condition="'$(__RootBinDir)'==''">$(ProjectDir)bin\</RootBinDir>
+  </PropertyGroup>
+
+  <!-- Output paths -->
+  <PropertyGroup>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(RootBinDir)obj\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(BaseIntermediateOutputPath)\$(BuildOS).$(BuildArch).$(BuildType)</IntermediateOutputPath>
+    <OutputPath Condition="'$(OutputPath)' == ''">$(BaseIntermediateOutputPath)\$(BuildOS).$(BuildArch).$(BuildType)</OutputPath>
+    <FinalOutputPath Condition="'$(FinalOutputPath)' == ''">$(BinDir)</FinalOutputPath>
+  </PropertyGroup>
+
+  <Import Condition="Exists('$(ToolsDir)BuildVersion.targets')" Project="$(ToolsDir)BuildVersion.targets" />
```

Comment:
[BuildVersion.targets provides](https://github.com/dotnet/buildtools/blob/704cb636b15bd4da3c6d7602631f116ff282eb41/src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets#L6) a value for `BuildVersionFile`, which is required by [Versioning.targets](https://github.com/dotnet/buildtools/blob/704cb636b15bd4da3c6d7602631f116ff282eb41/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L352-L357).

To generate that default, I needed `BaseIntermediateOutputPath` from `/dir.props`, so I brought over the full `Output paths` property group and a single entry from `Common properties`.